src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String clr = colorSupplier.getRandomColor();
+        int index = new Random().nextInt(100);
```

Comment:
All magic numbers in your code should be constants. How do you think what max number will be returned by `random.nextInt(100);`?